remote.go

```diff
@@ -137,6 +137,42 @@ var showCmd = &cobra.Command{
 	},
 }
 
+var setCmd = &cobra.Command{
+	Use:   "set [REMOTE] [Property] [value]",
+	Short: "Set details about remote.",
+	Long:  `Set details about the given remote.`,
+	Args:  cobra.MinimumNArgs(3),
+	Run: func(cmd *cobra.Command, args []string) {
+		// Ensure project initialized.
+		config, err := client.GetProjectConfigFromDisk()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		remote, found := config.GetRemote(args[0])
+		if found {
+			success := config.SetProperty(args[1], args[2])
+			if success {
+				println("Configuration has been updated.")
+				config.PrintConfigDetails()
+				printRemoteDetails(remote)
+
+			} else {
```

Comment:
What happens if I want to change the Inertia version in the config file?
```
inertia set my_vps version v0.2.0
```
But you're not actually updating something about `my_vps` - the argument doesn't actually mean anything here.

Perhaps a better command tree would be:
```
inertia set [config property] [config value]
```
and:
```
inertia [REMOTE] set [remote property] [remote value]
```

Reply:
This way you also won't have to do this double loop (check for presence of property in `config`, then check for it in `remote`)

Reply:
good point I was thinking of making two commands as well.